openstudiocore/src/model/SiteGroundTemperatureBuildingSurface_Impl.hpp

```diff
@@ -30,31 +30,7 @@ namespace detail {
 
   /** SiteGroundTemperatureBuildingSurface_Impl is a ModelObject_Impl that is the implementation class for SiteGroundTemperatureBuildingSurface.*/
   class MODEL_API SiteGroundTemperatureBuildingSurface_Impl : public ModelObject_Impl {
-    Q_OBJECT;
```

Comment:
@macumber Does the GUI still rely on these Q_PROPERTY? With the change to grid view, I don't know off hand.